build.sbt

```diff
@@ -93,7 +93,9 @@ val checker = (project in file(s"$appsFolder/checker"))
       "com.gu" %% "content-api-client-aws" % "0.7",
       "com.gu" %% "content-api-client-default" % capiClientVersion,
       "edu.stanford.nlp" % "stanford-corenlp" % "3.4",
```

Comment:
do you need to exclude xom here too?

Reply:
I still see xalan in the dependency tree 
![image](https://user-images.githubusercontent.com/10963046/206168903-e58cb17f-b7e3-44cc-af23-4640af655411.png)

Reply:
Thanks, yes. [985ffd2](https://github.com/guardian/typerighter/pull/213/commits/985ffd2ede4164712550a250b19f4910eb7f0945)